docs/pages/faq.rst

```diff
@@ -9,14 +9,17 @@ behavior in PRAW and reddit.
 FAQ
 ---
 
-How do I get a comment by ID?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+How do I get a submission with a predefined comment tree?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If you have a permalink to a comment you can use get_submission on the
-permalink, and then the first comment should be the desired one.
+permalink, and then the first comment should be the desired one. You can
+also do this by choosing a ``comment_root`` with ``submission_id``.
 
->>> s = r.get_submission('http://www.reddit.com/r/redditdev/comments/s3vcj/_/c4axeer')
->>> your_comment = s.comments[0]
+>>> s1 = r.get_submission('http://www.reddit.com/r/redditdev/comments/s3vcj/_/c4axeer')
```

Comment:
Great example for the comment tree case. However, I think we still want to maintain the question about getting a single comment as that really did come up frequently.